sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -411,4 +324,277 @@ public void run() {
       }
     }
   }
+
+  @Immutable
+  @AutoValue
+  public abstract static class Config {
+
+    public abstract boolean isReportOnlySampled();
```

Comment:
how about `exportOnlySampled()` as the name for this? `report` isn't a term that shows up anywhere else that I know of.

Reply:
I also found that strange when I look at the class the first time! Totally in favor of `exportOnlySampled()`.